p2p/querier.go

```diff
@@ -0,0 +1,62 @@
+package p2p
+
+import (
+	"context"
+	"time"
+
+	celestiatypes "github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/celestiaorg/orchestrator-relayer/types"
+)
+
+// QuerierI queries the P2P network for confirms.
+// Note: for now, we will be querying the P2P network directly without storing anything in the store.
+// this would make it easier to have a simple working version. Then, we can iterate from there on.
+type QuerierI interface {
```

Comment:
[not blocking nit]
The I at the end feels redundant and is usually not added unless there is something else that using the same name. If that's the case, then imho I think we should change one of the names so they don't match

Reply:
There will be the implementation of `QuerierI` called `Querier`, unless we find  a better name for it :cat: . I agree, the current naming scheme I have been following for these PRs is a bit dull

Reply:
okay, yeah its not the end of the world lol

on a unrelated note, why is the querier in the p2p package?

Reply:
there are multiple queriers, the tendermint querier (for getting the data commitments, the roots etc from tendermint) and the qgb module querier (for getting the attestations from the state machine) and the p2p querier (for eventually getting the attestations confirms)